common/horizon/etc/horizon.env

```
@@ -12,3 +12,4 @@ export DISABLE_ASSET_STATS="true"
 export HISTORY_ARCHIVE_URLS="__ARCHIVE__"
 export ADMIN_PORT=6060
 export PORT=8001
+export CHECKPOINT_FREQUENCY=__CHECKPOINT_SIZE__
```

Could we name the template token "frequency" instead of "size"? The term frequency is what we use in the config files, and using a single term to refer to a concept reduces load on readers. (See https://martinfowler.com/bliki/UbiquitousLanguage.html for context.)
```suggestion
export CHECKPOINT_FREQUENCY=__CHECKPOINT_FREQUENCY__
```

---

adjusted - https://github.com/stellar/quickstart/pull/414/commits/60745c69668a644952b20f595febebd9ef479e45